spec/lib/oauth/pre_authorization_spec.rb

```diff
@@ -107,43 +106,43 @@ module Doorkeeper::OAuth
       end
 
       it "accepts valid scopes" do
-        subject.scope = "public"
+        attributes[:scope] = "public"
         expect(subject).to be_authorizable
       end
 
       it "rejects (globally) non-valid scopes" do
-        subject.scope = "invalid"
+        attributes[:scope] = "invalid"
         expect(subject).not_to be_authorizable
       end
 
       it "rejects (application level) non-valid scopes" do
-        subject.scope = "profile"
+        attributes[:scope] = "profile"
         expect(subject).to_not be_authorizable
       end
 
       it "accepts scopes which are permitted for grant_type" do
         allow(server).to receive(:scopes_by_grant_type).and_return(authorization_code: [:public])
-        subject.scope = "public"
+        attributes[:scope] = "public"
         expect(subject).to be_authorizable
       end
 
       it "rejects scopes which are not permitted for grant_type" do
         allow(server).to receive(:scopes_by_grant_type).and_return(authorization_code: [:profile])
-        subject.scope = "public"
+        attributes[:scope] = "public"
         expect(subject).not_to be_authorizable
       end
     end
 
     it "uses default scopes when none is required" do
       allow(server).to receive(:default_scopes).and_return(Scopes.from_string("default"))
-      subject.scope = nil
-      subject.client = client
+      attributes[:scope] = nil
+      expect(subject).to be_authorizable
```

Comment:
The same with https://github.com/doorkeeper-gem/doorkeeper/pull/1300/files#r313330230